pages/docker_compose.md

```diff
@@ -61,6 +61,16 @@ If you are using Docker Machine on Mac or Windows, your Docker daemon has only l
 
 </div>
 
+
+If you encounter such an error `npm ERR! Error: EACCES: permission denied` when installing JHipster UML (or any unbundled package), your container may not have sudo (for instance, sudo isn't bundled with Ubuntu Xenial).
+A solution would be to do:
```

Comment:
That's what I proposed to the original poster of the issue in JUML's issue tracker. I'll add it as a note to my side note.